src/containers/ListingPage/ListingPage.module.css

```diff
@@ -699,6 +699,10 @@
   border: 0;
   cursor: pointer;
 
+  img {
```

Comment:
This can't use element styling¹ as any `<img>` element (after this code-chunk) is loaded would be affected.

That's why element styling happens in one place: marketplaceDefaults.css.

¹ CSS Modules can't make element-styles unique. It only affects classes.